build.gradle

```diff
@@ -28,6 +28,15 @@ task validateYamls(group: 'verification', description: 'Validates YAML files.')
 }
 
 subprojects {
+    apply plugin: 'checkstyle'
+    apply plugin: 'jacoco'
+    apply plugin: 'java'
+    apply plugin: 'net.ltgt.errorprone'
+
+    apply from: "${rootProject.projectDir}/gradle/scripts/release.gradle"
+    apply from: "${rootProject.projectDir}/gradle/scripts/remote-lib.gradle"
+    apply from: "${rootProject.projectDir}/gradle/scripts/version.gradle"
+
     group = 'triplea'
     sourceCompatibility = JavaVersion.VERSION_1_8
     targetCompatibility = JavaVersion.VERSION_1_8
```

Comment:
This was a regression from when I was consolidating the buildscripts.  The `sourceCompatibility` and `targetCompatibility` properties must be set _after_ the `java` plugin is applied, otherwise they will be overwritten with the version of the running JVM.